angular/demo/bootstrap/src/app/samples/accordion/customHeaders.route.ts

```diff
@@ -1,43 +1,34 @@
-import {AgnosUIAngularModule} from '@agnos-ui/angular';
+import {AgnosUIAngularModule, UseDirective} from '@agnos-ui/angular';
 import {Component} from '@angular/core';
 import BODY from '@agnos-ui/common/samples/accordion/body.txt';
 
 @Component({
 	standalone: true,
-	imports: [AgnosUIAngularModule],
+	imports: [AgnosUIAngularModule, UseDirective],
 	template: `
 		<div auAccordion #accordion="auAccordion">
 			<div auAccordionItem>
 				<ng-template auAccordionItemStructure let-state="state" let-widget="widget">
 					<div
-						class="{{ state.itemHeaderClass }} accordion-button accordion-header custom-header justify-content-between"
+						[auUse]="widget.directives.headerDirective"
+						class="accordion-button accordion-header custom-header justify-content-between"
 						[class.collapsed]="!state.itemVisible"
 						role="heading"
 						aria-level="2"
 					>
 						<p class="m-0">First panel - {{ state.itemVisible ? 'opened' : 'collapsed' }}</p>
 						<button
 							type="button"
-							id="{{ state.itemId }}-toggle"
-							(click)="widget.actions.click()"
-							class="btn btn-link p-0 {{ state.itemButtonClass }}"
-							[class.collapsed]="!state.itemVisible"
+							class="btn btn-link p-0 {{ state.itemButtonClass }} au-accordion-item-button"
 							[disabled]="state.itemDisabled"
```

Comment:
Why is the *disabled* attribute not managed by the toggleDirective ?

Reply:
because is part of the button directive, the idea is that you could use the toggle directive even on elements that are not buttons. It's coming from ng-bootstrap approach needed to showcase the custom header